# patch needed for providing the proper includes needed for
# gcc-4.3 (08/25/2008)

diff -Naur qtiplot-0.9.7/qtiplot/src/plot2D/ScaleEngine.cpp qtiplot-0.9.7.new/qtiplot/src/plot2D/ScaleEngine.cpp
--- qtiplot-0.9.7/qtiplot/src/plot2D/ScaleEngine.cpp	2008-06-12 13:19:20.000000000 -0400
+++ qtiplot-0.9.7.new/qtiplot/src/plot2D/ScaleEngine.cpp	2008-08-25 18:38:35.000000000 -0400
@@ -27,6 +27,7 @@
  *                                                                         *
  ***************************************************************************/
 #include "ScaleEngine.h"
+#include <limits.h>
 
 QwtScaleTransformation* ScaleEngine::transformation() const
 {
